### PR TITLE
Change historical to cancelled

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#17032

*Description of changes:*
- Changes historical to cancelled and adds tests
- Clear filters when returning to dashboard

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/tables/RegistrationTable.vue [hunk 1]
```diff
@@ -705,6 +706,11 @@ export default defineComponent({
       { deep: true }
     )
 
+    // Ensures filtering is cleared when returing to dashboard from registrations
+    onBeforeMount(() => {
+      clearFilters()
+    })
+
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/RegistrationTable.vue:713]: Is this a request by design?

**Review by cameron-eyds — APPROVED** [review]
LGTM!

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1485-14w86ipi.web.app

File: ppr-ui/src/components/tables/RegistrationTable.vue (hunk 1 above)
Comment (orelbn) [reply ppr-ui/src/components/tables/RegistrationTable.vue:713]: No I can remove this, but I thought it was strange behaviour that when you return from a registration at the moment, it leaves the filter columns showing even though it is not filtering.

image

@tlebedovich is my assumption of the expected behaviour accurate?

`Clear filters when returning to dashboard`

Comment (cameron-eyds) [reply ppr-ui/src/components/tables/RegistrationTable.vue:713]: Ah i see. 
Yea, i think that makes sense to clear, the above scenario is definitely confusing.

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Comment by tlebedovich** [comment]
@orelbn I think it make sense for now to clear the filters if it indeed is not actually filtering. But I will check with scott in terms of how this should actually be working and can create a ticket to make further updates in terms of when/if to save the table filtered states.

**Pull request merged**